src/main/java/friday/Task.java

```diff
@@ -0,0 +1,51 @@
+package friday;
+
+/**
+ * Represents a task in the Friday application.
+ */
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a status icon based on whether the task is done.
+     * @return A string representing the status icon.
```

Comment:
Should the Javadocs have space between the comments and the @ tags?

```suggestion
     * Returns a status icon based on whether the task is done.
     *
     * @return A string representing the status icon.
```